pom.xml

```diff
@@ -124,6 +124,28 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.0.1</version>
+                <executions>
+                    <execution>
+                        <id>aggregate-javadoc</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>aggregate-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
```

Comment:
Per the comment left by @anthonyvdotbe, this <configuration> block could be moved into the <execution> block above and then it would only apply to that "aggregate-javadoc" execution.  That might avoid potential conflicts with any component attempting to use this pom as a parent.